ledger-service/http-json/src/itlib/scala/http/AbstractWebsocketServiceIntegrationTest.scala

```diff
@@ -59,6 +60,12 @@ abstract class AbstractWebsocketServiceIntegrationTest(val integration: String)
     with AbstractHttpServiceIntegrationTestFuns
     with BeforeAndAfterAll {
 
+  // Guard tests which use features that are not available in Canton Community Edition
+  private implicit final class EditionBranchingSupport(private val label: String) {
+    def ifSupportsPruning_-(fn: => Unit): Unit =
+      if (edition.Canton.supportsPruning) label - fn else ()
```

Comment:
I don't understand this `label - fn`. Can you explain ?

Reply:
It's just using the dash operator from `AnyFreeSpec` to register a labelled test.

https://www.scalatest.org/scaladoc/3.2.16/org/scalatest/freespec/AnyFreeSpec.html